### DISCUSSION TITLE
[FR] Slider in Cover card

### DISCUSSION THREAD

**Discussion by juanpa669** [Clooos/Bubble-Card#316] [Feature requests]
Add an optional slider in cover card to set cover position.

**Comment by itsayopapi** [comment]
@juanpa669 can i work on this issue?

**Comment by Clooos** [comment]
Hi @itsayopapi, I'm working on a huge code refactoring for v1.6.0, you should probably wait for that version.

But thanks a lot for the suggestion!

**Comment by RiderCrazy** [comment]
Hi,

Will this features be added ?
I use mushroom for this for now.

![image](https://github.com/user-attachments/assets/6c33113b-2e1f-4e79-b1c9-5a06f2efed9d)
